### PR TITLE
Nandini

### PR DESCRIPTION
**Pull request by Nandiniverma2024**

### CODE REVIEW

File: README.md
Comment (tanishdotdev) [inline README.md:1]: Update this according to the changes done in headings and paragraphs.

File: src/atoms/atoms-nandini/headings/one/5xl/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/headings/one/5xl/index.astro:1]: It should be src/atoms/atoms-nandini/headings/one/5xl/white/index.astro

File: src/components/home/lets-talk/index.astro
Comment (tanishdotdev) [inline src/components/home/lets-talk/index.astro:1]: 1. Remove text-center from div.
2. Add links to buttons.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

**Comment by Nandiniverma2024** [comment]
Done

**Pull request closed without merging**